### PR TITLE
Quadrat/update font sizes

### PR DESCRIPTION
**Pull request by jeffikus**
#### Changes proposed in this Pull Request:

Updates the font sizes to match the Design file changes from @beafialho 

#### Related issue(s):

None.

#### Test Plan:

**Test that the font sizes on Desktop are:**
h1: 80px
h2: 65px
h3: 38px
h4: 22px
h5: 20px
h6: 18px
P Tiny: 16px
P Small: 18px
P Normal: 20px
P Large: 22px
P Extra Large: 38px
P Huge: 48px
Site Title: 20px
Site Tag: 16px

**Test that the font sizes on Mobile are:**
h1: 48px
h2: 36px
h3: 28px
h4: 22px
h5: 20px
h6: 18px
P Tiny: 16px
P Small: 18px
P Normal: 20px
P Large: 22px
P Extra Large: 28px
P Huge: 36px

Test that Quote Citation font size is: 16px
Test that Query Pagination font size is: 20px

### CODE REVIEW

File: quadrat/child-theme.json [hunk 1]
```diff
@@ -278,7 +278,7 @@
 			},
 			"core/navigation": {
 				"typography": {
-					"fontSize": "18px"
+					"fontSize": "20px"
```
Comment (scruffian) [inline quadrat/child-theme.json]: This should use the variable `var(--wp--preset--font-size--normal)`

File: quadrat/child-theme.json [hunk 2]
```diff
@@ -306,7 +306,7 @@
 				},
 				"citation": {
 					"typography": {
-						"fontSize": "15px",
+						"fontSize": "16px",
```
Comment (scruffian) [inline quadrat/child-theme.json]: This should use the variable var(--wp--preset--font-size--tiny)

File: quadrat/child-theme.json [hunk 3]
```diff
@@ -349,7 +349,7 @@
 			},
 			"core/query-pagination": {
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "20px",
```
Comment (scruffian) [inline quadrat/child-theme.json]: This should use the variable var(--wp--preset--font-size--normal)

File: quadrat/child-theme.json [hunk 4]
```diff
@@ -358,7 +358,7 @@
 					"link": "var(--wp--custom--color--primary)"
 				},
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "20px",
```
Comment (scruffian) [inline quadrat/child-theme.json]: This should use the variable `var(--wp--preset--font-size--normal)`

File: quadrat/child-theme.json [hunk 5]
```diff
@@ -389,21 +389,21 @@
 			"h4": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "20px",
+					"fontSize": "22px",
```
Comment (scruffian) [inline quadrat/child-theme.json]: This should use the variable `var(--wp--preset--font-size--large)`

File: quadrat/child-theme.json [hunk 6]
```diff
@@ -389,21 +389,21 @@
...
 					"lineHeight": 1.4
 				}
 			},
 			"h5": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "18px",
+					"fontSize": "20px",
```
Comment (scruffian) [inline quadrat/child-theme.json]: This should use the variable `var(--wp--preset--font-size--normal)`

File: quadrat/child-theme.json [hunk 7]
```diff
@@ -389,21 +389,21 @@
...
 					"lineHeight": 1.4
 				}
 			},
 			"h6": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "16px",
+					"fontSize": "18px",
```
Comment (scruffian) [inline quadrat/child-theme.json]: This should use the variable `var(--wp--preset--font-size--small)`

File: quadrat/sass/blocks/_navigation.scss [hunk 8]
```diff
@@ -45,7 +45,7 @@
 						.wp-block-pages-list__item__link,
 						.wp-block-navigation-link__content {
 							padding: 0;
-							font-size: 15px;
+							font-size: 16px;
```
Comment (scruffian) [inline quadrat/sass/blocks/_navigation.scss]: This should use the variable `var(--wp--preset--font-size--tiny)`

File: quadrat/child-theme.json (hunk 1 above)
Comment (jeffikus) [reply quadrat/child-theme.json]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/child-theme.json (hunk 2 above)
Comment (jeffikus) [reply quadrat/child-theme.json]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/child-theme.json (hunk 3 above)
Comment (jeffikus) [reply quadrat/child-theme.json]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/child-theme.json (hunk 4 above)
Comment (jeffikus) [reply quadrat/child-theme.json]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/child-theme.json (hunk 5 above)
Comment (jeffikus) [reply quadrat/child-theme.json]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/child-theme.json (hunk 6 above)
Comment (jeffikus) [reply quadrat/child-theme.json]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/child-theme.json (hunk 7 above)
Comment (jeffikus) [reply quadrat/child-theme.json]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/sass/blocks/_navigation.scss (hunk 8 above)
Comment (jeffikus) [reply quadrat/sass/blocks/_navigation.scss]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

File: quadrat/sass/blocks/_navigation.scss [hunk 9]
```diff
@@ -45,7 +45,7 @@
 						.wp-block-pages-list__item__link,
 						.wp-block-navigation-link__content {
 							padding: 0;
-							font-size: 15px;
+							font-size: var(--wp--preset--font-size--tiny);
```
Comment (scruffian) [inline quadrat/sass/blocks/_navigation.scss:48]: There's a font size declaration in line 31 of this file too..

Comment (jeffikus) [reply quadrat/sass/blocks/_navigation.scss:48]: Fixed in https://github.com/Automattic/themes/pull/4683/commits/2532bae61948e88efce92f0cb2b3f7adf7d75527 and did a sweep in the theme for other areas of font-size in https://github.com/Automattic/themes/pull/4683/commits/51266f3eea9ac012a6606ccf601ef253d6f68976

**Comment by scruffian** [comment]
This is looking good. We need to make the same changes to geologist.

**Comment by jeffikus** [comment]
Yes, I'll get cracking on the changes to Geologist later today as well. Thanks for the review!

**Review by scruffian — APPROVED** [review]

**Comment by scruffian** [comment]
Deployed in D67302-code

**Pull request merged**